content/sigs/docs/gsod/2021/organization-proposal.adoc

```diff
@@ -36,15 +34,63 @@ The improved documentation will also be referenced in a Kubernetes Solutions Pag
 
 === Your project's problem
 
-More details to be inserted here...
+Jenkins users have significant challenges deploying on Kubernetes.
+A successful Jenkins deployment on Kubernetes combines many different Jenkins components.
+Mistakes in using those components create a less reliable and less usable Jenkins installation.
+
+There are many levels to be addressed and many places where incomplete information and mistakes lead to installation and upgrade problems.
+The Jenkins documentation created by this project will describe the compromises and the alternatives for Jenkins on Kubernetes.
+Documentation users will be able to confidently deploy Jenkins on Kubernetes with an understanding of the strengths and weaknesses of their solution.
 
 === Your project's scope
 
-More details to be inserted here...
+Project scope will be adapted based on available time from the writer.
+
+==== Preparation Phase
+
+* Review link:https://leanpub.com/the-devops-2-3-toolkit[DevOps 2.3 Toolkit] by Viktor Farcic
```

Comment:
Repo is here: https://github.com/vfarcic/devops24

Reply:
Added as a link in fe277d19708090ac2b24ae7a4b12ad4ef4da37d8